components/body/index.jade

```diff
@@ -57,11 +57,11 @@
       .right-gutter-formatter
         p.body-section-sub-header= copy.partnership.gallery.subTitle[language]
         p.format-spacing= copy.partnership.gallery.text[language]
-        button.avant-garde-button-black= copy.partnership.gallery.link[language]
+        button.avant-garde-button-black(onclick="location.href='https://www.artsy.net/galleries';")= copy.partnership.gallery.link[language]
```

Comment:
@broskoski I did this because I had a selector that was selecting all anchor tags that are children to `section`. In addition I was having difficulty overwriting some of the styling coming in from the elan library. I can switch the selector to be `.chevron-nav-list section > a:first-child` but I'm not sure the best approach for overwriting the library's styling. Suggestions?